devstack/lib/nfp

```diff
@@ -272,6 +272,8 @@ function create_nfp_gbp_resources {
         gbp service-profile-create --servicetype FIREWALL --insertion-mode l3 --shared True --service-flavor service_vendor=vyos,device_type=nova --vendor NFP fw_profile
         gbp service-profile-create --servicetype VPN --insertion-mode l3 --shared True --service-flavor service_vendor=vyos,device_type=nova --vendor NFP vpn_profile
         
+        create_ext_net
+        create_ep_and_nsp
         gbp l3policy-create --ip-version 4 --proxy-ip-pool=192.169.0.0/24 --ip-pool 120.0.0.0/24 --subnet-prefix-length 24 service_management
```

Comment:
Can we make these statements common for both advanced and enterprise modes. Also add floating IP address allocation for visibility VM outside the create of service mgmt ptg.